app/api/v1/endpoints/tenants.py

```
@@ -37,7 +39,7 @@
 async def create_tenant(
     request: Request,
     body: TenantCreateRequest,
-    backend: Annotated[DbOrFirestore, Depends(_get_db_or_firestore_write)],
+    db: Annotated[AsyncSession, Depends(get_db_transactional)],
     tenant_svc: TenantCreationService = Depends(get_tenant_creation_service),
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)